tests/integration_tests.rs

```diff
@@ -111,6 +111,28 @@ async fn test_unlogged() {
     assert_eq!(message.msg_id, msg_id);
 }
 
+// Integration tests are ignored by default
+#[ignore]
+#[tokio::test]
+async fn test_max_queue_name_size() {
+    let conn = init_database().await;
+
+    // CREATE with default retention and partition strategy
+    let too_big = 'a'.repeat(48);
+    let result1 = sqlx::query(&format!("SELECT {PGMQ_SCHEMA}.create('{too_big}');"))
+        .execute(&conn)
+        .await;
+
+    assert!(result1.is_err());
+
+    let okay = 'a'.repeat(47);
+    let result2 = sqlx::query(&format!("SELECT {PGMQ_SCHEMA}.create('{okay}');"))
+        .execute(&conn)
+        .await;
+
+    assert!(result2.is_ok());
```

Comment:
Would be awesome if we could capture notices here somehow, as then we would be able to fail the test if postgres truncated some identifier.